active/dceu.md

```diff
@@ -20,20 +27,22 @@ Delegated responsibilities TBD.
                          
 ## Future membership
 
-Membership is open to former organizers of DjangoCon EUs. Questions about membership can be directed to european-organizers-support@djangoproject.com. [Contact the board](https://www.djangoproject.com/contact/foundation/) to express interest.
+Membership is open to former organizers of DjangoCon EUs. Questions about membership can be directed to european-organizers-support@djangoproject.com.
 
 Membership is self-managed: new members may self-nominate; the WG will vote (50%+1) to approve/deny new members; the WG will directly vote on new Chair/Co-Chairs.
 
 ## Budget
 
-TBD
+No budget needed at this time.
 
 ## Comms
 
-The team will operate via a Google Group, reachable via european-organizers-support@djangoproject.com. There might be additional more synchronous communication (e.g. Slack or Discord) if needed.
+The team internally operates via a Google Group. This is also the primary external point of contact to reach the working group: european-organizers-support@djangoproject.com
+
+The working group is also present in the #europe channel on DSF slack as a public place of discussion.
 
 The team will also set up a collection of data and information, in something like a Google Drive folder. A static informational website might follow at a later date.
 
 ## Reporting
 
-TBD
+As DjangoCon Europe has a native yearly cycle, the working group will formally report to the board once per year. However, since the working group is not making any decisions and does most of its work by advising the board, the board is expected to be updated on important DjangoCon Europe matters in real-time througout the year.
```

Comment:
To me as a board member yearly isn’t frequent enough. My expectation would be monthly or quarterly. I’d rather receive short but frequent reports, as otherwise I could be out of the loop on what the group is up to for a while. A yearly cycle sounds great to me to make a public report of the group’s activities, but for the board I’d like to know about progress on:

1. Our selection process. Both the methodology, and practically where we’re at with different proposals / EoIs.
2. How the group works with this year’s organizers.
3. Our organizational knowledge improvements.

We also use reporting as a way to keep track of group health (see [Shutting down WGs](https://github.com/django/dsf-working-groups#shutting-down-wgs)).

Reply:
Looking at the months the WG has existed now, there was not a single piece of "action" within the WG (like when vetting the 2025 proposals or giving feedback on the form) where no board member (you, Cagil, Chaim) was deeply involved. I assume that is going to stay that way since

- the board formally receives the proposals
- the board makes the decisions in the selection process

So at least for item 1 of your list, if I would now send a report for this quarter that says "Becky worked with Thibauld to vet the proposals for 2025", that did not appear incredible useful to me.

But if you prefer, we can just change it to a monthly cycle, especially for the "How the group works with this year’s organizers" part it makes sense. Although I assume that, since the 2025 team has prior experience, they will not need the support from us very much, but I'm okay with sending a sometimes-almost-empty email once a month.

Reply:
Let’s see what others think. From my perspective, if you think there’s not much to report on, a report of "there isn’t much to report on" is completely fine. We can always ask for more info.

> there was not a single piece of "action" within the WG […] where no board member was deeply involved

Board members don’t always report on their activities with the rest of the board so I don’t think that’s relevant?

Reply:
Just noting without further feedback, I’m happy to move forward with the existing wording :)